client/gulpfile.js

```diff
@@ -67,6 +103,7 @@ gulp.task('watch', () => {
   })
   gulp.watch(sources.css.all, ['styles', 'stylelint'])
   gulp.watch(sources.scripts.all, ['scripts'])
+  gulp.watch(sources.images.svg.all, ['svgstore'])
```

Comment:
An example would be if you want to add a build script to run on all images, but not SVGs, you would then have to have an exclude in that script, which is the unneeded complexity I am suggesting at.